test/utils/sequelize-helper.js

```diff
@@ -0,0 +1,51 @@
+const Sequelize = require('sequelize');
+const fs = require('fs');
+const path = require('path');
+
+class SequelizeHelper {
+  connect(url) {
+    this.sequelize = new Sequelize(url, {
+      logging: false,
+      pool: { maxConnections: 10, minConnections: 1 },
+      dialectOptions: {
+        multipleStatements: true,
+      },
+    });
+    return new Promise((resolve, reject) => {
+      this.sequelize.authenticate()
+        .then(() => resolve(this.sequelize))
+        .catch(err => reject(err));
+    });
+  }
+
+  async given(tableName) {
```

Comment:
[IMO] I would use fixture term. `given a fixture` make more sense to me.

Reply:
In that particular case, it only accepts a table name (my `given` functions works actually only with a table name). That's far from perfect, but I felt like it's OK.